### ISSUE TITLE
Kill quest objective credit received while in spirit form

### ISSUE THREAD

**Issue by ognennyy** [LightsHope/server#1055]
At approximately 03:35 (server time) I received credit for a kill of a Bloodfury Slayer - quest "Harpies Threaten in the Stonetalon Mountains - while in spirit form.

Before the incident I had 1/7 kills credited for the Slayers.  While engaged in PvP combat with a Night Elf Hunter and Night Elf druid, I prox agrod a Slayer.  Then my frost nova tagged her.  Members of the horde attempted to help me in my fight.  I was killed.  I released my spirit and while running back to my corpse, I was given credit for the death of a Slayer, giving me 2/7 kill credits.

**Comment by Cabro** [comment]
Oh this really shouldn't happen.

**Comment by berillis** [comment]
Of course it should happen, but the important part is the range where you can get the credit.

**Comment by Cabro** [comment]
No, it shouldn't happen. You should only get quest objective completed if you are dead and not released. If you are a ghost, then you shouldn't get quest completion/xp/rep.
